src/openzaak/components/zaken/tests/test_zaken.py

```diff
@@ -574,3 +577,131 @@ def test_zaak_archiefactiedatum_afleidingswijze_ingangsdatum_besluit(self):
         self.assertEqual(
             zaak.archiefactiedatum, date(2030, 5, 3)  # 2020-05-03 + 10 years
         )
+
+
+@tag("external-urls")
+class ZaakCreateExternalURLsTests(JWTAuthMixin, APITestCase):
+    heeft_alle_autorisaties = True
+    list_url = get_operation_url("zaak_create")
+
+    def test_create_external_zaaktype(self):
+        catalogus = "https://externe.catalogus.nl/api/v1/catalogussen/1c8e36be-338c-4c07-ac5e-1adf55bec04a"
+        zaaktype = "https://externe.catalogus.nl/api/v1/zaaktypen/b71f72ef-198d-44d8-af64-ae1932df830a"
+
+        with requests_mock.Mocker(real_http=True) as m:
```

Comment:
why the `real_http`? I'd prefer the test to error if a URL is fetched that accesses the real network. Those calls should be mocked.

Reply:
In the `LooseFkResourceValidator` `ZTC_API_SPEC` is requested. I thought we use it in tests without mocking it. And the tests for `Besluit.besluittype` also don't mock it.

Reply:
ah yeah... now I remember that we have some calls going through.

I'll create a new issue so we can leave it as it is. API spec mocking is annoying